app/javascript/react-toolbar/index.jsx

```diff
@@ -0,0 +1,12 @@
+import ReactDOM from 'react-dom';
+import {Credentials} from './new-credentials';
+
+var subscriber = ManageIQ.extensionComponents.subscribe('toolbar');
+subscriber.with(createNewFormFields);
+
+function createNewFormFields(component) {
+    setTimeout(() => {
+      let element = component.render.addNewButton();
```

Comment:
Shouldn't `addNewButton` be called something like `getNewButtonContainer` or similar?

Reply:
It's just an example on how to use the extensible component, I guess Martin will change it a bit since he does not want to replace whole first section with one button.